lib/Dialect/FIRRTL/Transforms/GrandCentralSignalMappings.cpp

```diff
@@ -128,7 +258,7 @@ void ModuleSignalMappings::run() {
   // Check whether this module has any `SignalDriverAnnotation`s. These indicate
```

Comment:
Maybe update this comment (and the one a few lines above), to reflect the new module annotation?

Reply:
Thanks for checking this. Documentation is much easier to keep up to date when making the modifications. 😅